util/util.go

```diff
@@ -132,3 +136,26 @@ func IntV(in *int) int {
 	}
 	return *in
 }
+
+// FSGeneration generates a name and a path that will be used to feed
+// the `fsMap` in case consumer provided an empty map.
+// FSMap is required to link the filesystem from v2 to v3 with the rest of the configuration.
+func FSGeneration(name string, fsMap map[string]string) error {
+	if len(name) == 0 {
+		rand.Seed(time.Now().Unix())
+		generatedName := rand.Int()
+		name = strconv.Itoa(generatedName)
```

Comment:
```suggestion
		name = "ignition" + strconv.Itoa(generatedName)
```
Maybe it's nice to give it a "namespace"

Reply:
Yep, that's right. Good suggestion !